app/assets/src/components/views/discovery/EmptyStateModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { withAnalytics } from "~/api/analytics";
+import BacteriaCultureIcon from "~ui/icons/BacteriaCultureIcon";
+import Link from "~ui/controls/Link";
+import Modal from "~ui/containers/Modal";
+
+import cs from "./empty_state_modal.scss";
+import InfoBanner from "./InfoBanner";
+
+const EmptyStateModal = ({ onClose }) => {
+  return (
+    <Modal
+      open
+      narrow
+      tall
+      className={cs.emptyStateModal}
+      onClose={withAnalytics(onClose, "EmptyStatesModal_close-modal_clicked")}
+    >
+      <InfoBanner
+        className={cs.emptyStateBanner}
+        icon={<BacteriaCultureIcon />}
+        iconClassName={cs.iconClassName}
+        message={
+          <React.Fragment>
+            To get started, you can{" "}
+            <Link className={cs.link} href={"/samples/upload"}>
```

Comment:
why not put analytics tracking on all links? 

also I'm wondering whether now is a good time to put the `cs.link` color into `Link` as a default. it seems a little crazy to define the link color in every instance of `Link`.

Reply:
> why not put analytics tracking on all links?

Yes, i thought of that but end up forgetting, will put it on the component.

> is a good time to put the cs.link color into Link as a default

Seems like we should do it but it is a bit risky... Will take a look